lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -66,13 +125,16 @@ function CFCRandomSpawn.handlePlayerSpawn( ply )
     if not ( ply and IsValid( ply ) ) then return end
     if IsValid( ply.LinkedSpawnPoint ) then return end
 
```

Comment:
Im not sure why `ply.isCFCRespawning` is needed here, it's set and unset in the same tick? cant you pass this player through function arguments and get the same result?